### PR TITLE
[531] refactoring  for removing Antd

### LINKED ISSUE #531: Refactor components from Ant to MUI

### PR DESCRIPTION
**Pull request by xdemocle**
**Detailed description**:

**Which issue(s) this PR fixes**:
Fixes #531 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: src/design-system/Modal/modal-styles.ts [hunk 1]
```diff
@@ -31,9 +30,9 @@ const StyledBackdrop = styled('div')({
 
 export const ModalBoxStyled = styled(Box)({
   width: 'auto',
-  maxHeight: 858,
+  maxHeight: 800,
   position: 'relative',
-  backgroundColor: THEME_COLORS.darkBlue03,
+  background: 'var(--theme-card-color)',
```
Comment (xdemocle) [inline src/design-system/Modal/modal-styles.ts:35]: this is not okay.

Into folder desygn-system we use the vars THEME_COLORS.darkBlue03 declared in src/themes/theme-constants.ts.

check  src/themes/theme-constants.ts yo uwill understand

File: src/modules/scene-builder/components/scene-builder-profile/index.tsx [hunk 2]
```diff
@@ -83,7 +83,9 @@ const SceneBuilderProfile: FC<ISceneBuilderProfile> = ({ builder }) => {
             <TypographyStyled variant="h4" style={{ textTransform: 'uppercase' }}>
               My Website
             </TypographyStyled>
-            <a href={builder?.website}>{builder?.website}</a>
+            <a style={{ color: 'var(--theme-accent-color)' }} href={builder?.website}>
```
Comment (xdemocle) [inline src/modules/scene-builder/components/scene-builder-profile/index.tsx:86]: remove also this. I fixed all a links in global style

File: src/components/custom/error-boundary/index.tsx [hunk 3]
```diff
@@ -37,30 +30,352 @@ export default class ErrorBoundary extends Component<any, State> {
...
+                  mask="url(#d)"
+                ></path>
+                <path
+                  d="M181.31 293.666h-27.712a3.209 3.209 0 0 1-3.209-3.21V269.79a3.209 3.209 0 0 1 3.209-3.21h27.711a3.209 3.209 0 0 1 3.209 3.21v20.668a3.209 3.209 0 0 1-3.209 3.209"
+                  fill="#F2D7AD"
+                  mask="url(#d)"
+                ></path>
+              </g>
+            </svg>
+          </div>
```
Comment (failfmi) [inline src/components/custom/error-boundary/index.tsx:360]: What exactly are those?

**Review by failfmi — CHANGES_REQUESTED** [review]
The claim modal is broken.

Also, you can compare this version running locally with the latest on develop, and you will notice that some styles might have changed.

File: src/components/custom/error-boundary/index.tsx (hunk 3 above)
Comment (xdemocle) [reply src/components/custom/error-boundary/index.tsx]: moved the svg outside

**Comment by xdemocle** [comment]
> The claim modal is broken.
> 
> Also, you can compare this version running locally with the latest on develop, and you will notice that some styles might have changed.

Modals have different style now according to Design on Figma. Before they were wrong now they are aligned to Figma that are the same used for metaportal coming from default styling of design-system package

**Comment by xdemocle** [comment]
@failfmi  The claim modal was also checked and fixed. What other specific UI mistakes you see in that modal?

**Comment by xdemocle** [comment]
> Price icons on the small cards are padded to the price as well

this fixed

**Comment by xdemocle** [comment]
> The claim modal is broken.
> 
> Also, you can compare this version running locally with the latest on develop, and you will notice that some styles might have changed.

ok also claim modal is fixed

**Review by failfmi — APPROVED** [review]

**Pull request merged**